docs/getting-started/zowe-architecture.md

```diff
@@ -4,4 +4,57 @@ Zowe&trade; is a collection of components that together form a framework that al
+
+## zssServer
+
+The Zowe desktop delegates a number of its services to the zssServer which it accesses through the http port 8542.  The zssServer is written in metalC and has native calls to z/OS to provide its services.  The zssserver logs are written to `<INSTANCE_DIR>/logs/zssServer-yyyy-mm-dd-hh-mm.log`. 
+
+## API Gateway
+
+The API Gateway is a proxy server that routes requests from clients on its northbound edge, such as web browsers or the Zowe command line interface, to servers on its southbound edge that are able to provide data to serve the request.  It is also responsible for generating the authentication token used to provide single sign-on (SSO) functionality.  The API Gateway homepage is `https://<ZOWE_HOST_IP>:7554`, that after authentication allows you to navigate to the API Catalog. 
+
+<img src="../images/api-mediation/api-mediationlayer.png" alt="Zowe API Mediation Layer" width="600px"/> 
+
+## API Catalog
+
+The API Catalog provides a list of the API services that have registered themselves as catalog tiles.  These allow you to view the available APIs from Zowe's southbound servers as well as test REST API calls.  
+
+<img src="../images/api-mediation/api-catalog.png" alt="Zowe API Catalog" width="600px"/> 
+
+## API Discovery
+
+The API Discovery server acts as the registration service broker between the API Gateway and its southbound servers.  It can be accessed through the URL `https://<ZOWE_HOST_IP>:7552`.  The API discovery homepage lets you view the list of registered API services.  
+
+<img src="../images/api-mediation/api-discovery.png" alt="Zowe API Discovery" width="600px"/> 
+
+## MVS, JES and USS UI
+
+Zowe provides a number of rich GUI web applications for working with z/OS.  This includes the MVS Explorer for data sets, the JES Explorer for jobs, and the USS Explorer for the Unix File System.  These can be accessed through the Zowe desktop.
+
+<img src="../images/mvd/desktop-explorers.png" alt="Zowe Desktop Explorers" width="600px"/> 
+
+### File API and JES API
+
+The File API server provides a set of REST APIS for working with z/OS data sets and Unix files.  These APIs are used by the MVS and USS Explorer apps.  
+
+The JES API server provides a set of REST APIs for working with JES.  These APIs are used by the JES Explorer app.
+
+Both the File API and JES API servers are registered as tiles on the API catalog, so developers can view their swagger definition as well as test API requests and responses.
+
+## Cross Memory server
+
+Unlike all of the servers described above which run under the `ZWESVSTC` started task as address spaces for USS processes, the cross memory server has its own separate started task `ZWESISTC` and its own user ID `ZWESIUSR` that runs the program `ZWESIS01`.  
```

Comment:
"cross memory" should be "cross-memory" in my book, but hyphenation is often omitted these days.  Up to you.